src/components/depth-player.js

```diff
@@ -227,12 +227,12 @@ export default class DepthPlayer extends EventEmitter {
 
   loadPropsFromObject(object) {
     // Update the shader based on the properties from the JSON
-    if (!object.textureWidth || !object.textureHeight) {
-      this.material.uniforms.width.value = object.depthImageSize.x;
-      this.material.uniforms.height.value = object.depthImageSize.y * 2;
-    } else {
+    if (object.textureWidth || object.textureHeight) { 
```

Comment:
The logical inverse of the previous statement would use an `&&`, not an `||`. Now this is prone to error if you only set one of the parameters.

Reply:
Thanks! I will split width and height to separate conditionals that way it's safer in case one of them exists